tests/unit/event_manager/adapters/conftest.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import pytest
+
+from ted_sws.event_manager.adapters.log.logger_mongo_handler import BufferedMongoHandler, MongoHandler
+
+
+@pytest.fixture
+def mongo_handler(mongodb_client):
```

Comment:
if you use a real Mongo client? then move the tests into `e2e` section, not `unit` section of tests.